otelcol-builder.datasetexporter-latest.yaml

```diff
@@ -210,4 +210,4 @@ replaces:
   - github.com/mattn/go-ieproxy => github.com/mattn/go-ieproxy v0.0.1
   # Needed to include latest version of our plugin
   - github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datasetexporter v0.83.0 => github.com/scalyr/opentelemetry-collector-contrib/exporter/datasetexporter datasetexporter-latest
-  - github.com/scalyr/dataset-go v0.12.1 => ./../dataset-go
```

Comment:
👍

Reply:
I wonder if we could add a check which verifies that the versions are matching here - https://github.com/scalyr/opentelemetry-exporter-dataset/blob/main/.github/workflows/release-image.yaml#L133

Reply:
I think something like this should go it - https://github.com/scalyr/opentelemetry-exporter-dataset/pull/15